agent-packaging/Dockerfile

```diff
@@ -0,0 +1,25 @@
+# Copyright 2018 Google Inc. All Rights Reserved.
```

Comment:
this stuff belongs in `guest-test-infra`

Reply:
as discussed offline, let's keep this here for now and i will do the refactoring later